openshift-hack/e2e/annotate/annotate.go

```diff
@@ -206,7 +206,8 @@ func (r *ginkgoTestRenamer) generateRename(name string, node types.TestSpec) {
 		}
 	}
 
-	if !r.excludedTestsFilter.MatchString(newName) {
+	// Append suite name to test, if it doesn't already have one
+	if !r.excludedTestsFilter.MatchString(newName) && !strings.Contains(newName, "[Suite:") {
```

Comment:
I bet this will drop out a bunch.  Perhaps a new tag like: `SpecialSuite:`?

Reply:
Only 2.  This change when vendored in origin drops out the etcd scale test, and `Image signature workflow can push a signed image to openshift registry and verify it,` which I was going to fix in origin by adding serial back to it.